pkg/api/types.go

```diff
@@ -450,10 +452,81 @@ type StepConfiguration struct {
 // if no explicit output tag is provided, the name
 // of the image is used as the tag.
 type InputImageTagStepConfiguration struct {
+	InputImage `json:",inline"`
+	Sources    *[]ImageStreamSource `json:"-"`
```

Comment:
Why is this a pointer to a slice?

Reply:
Per our IM convo, I think this was me getting mixed up with some of the slice mutation things we talked about re: the slice headers. Going to try to change this so that the `config` on `inputImageTagStep` is a pointer so that when that config is mutated, it is reflected in any step that holds that config.